rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -831,6 +835,23 @@ public TransactionReceiptDTO eth_getTransactionReceipt(TxHashParam transactionHa
         return new TransactionReceiptDTO(block, txInfo, signatureCache);
     }
 
+    @Override
+    public TransactionResultDTO[] eth_pendingTransactions() {
+        // get pending txs
+        List<Transaction> pendingTransactions = web3InformationRetriever.getTransactions("pending");
```

Comment:
we also could make `Web3InformationRetriever.PENDING` public and use it here as a constant

Reply:
yep, makes sense. just pushed it